.circleci/config.yml

```diff
@@ -19,10 +19,10 @@ jobs:
           command: npm run install-shields
 
       - run:
-          name: Run tests
+          name: Run core tests
           environment:
             mocha_reporter: mocha-junit-reporter
-            MOCHA_FILE: junit/app/results.xml
+            MOCHA_FILE: junit/core/results.xml
```

Comment:
I think this is running core plus frontend plus package, right?

Reply:
D'oh! Yeah you're right. The label `app` just didn't feel right to me when I added the test summary. Maybe `core-and-frontend`? 

Or maybe just leave it as is? 😆

Reply:
Main?

Reply:
I like that better than app, but does it carry a connotation that services aren't main?

Reply:
I'm okay with that. Though open to other suggestions!